test/Draw.rb

```diff
@@ -295,13 +291,27 @@ def test_composite
       Magick::SrcInCompositeOp,
       Magick::SrcOutCompositeOp,
       Magick::SrcOverCompositeOp,
-      Magick::SubtractCompositeOp,
       Magick::ThresholdCompositeOp,
       Magick::UndefinedCompositeOp,
       Magick::VividLightCompositeOp,
       Magick::XorCompositeOp
     ]
-    composite_operators << Magick::HardMixCompositeOp if Gem::Version.new('6.8.9') <= Gem::Version.new(IM_VERSION)
+    composite_operators << Magick::HardMixCompositeOp if IM_689
+    if IM_7
+      composite_operators << Magick::AlphaCompositeOp
+      composite_operators << Magick::CopyAlphaCompositeOp
+      composite_operators << Magick::DivideDstCompositeOp
+      composite_operators << Magick::MinusDstCompositeOp
```

Comment:
Can any of these be aliased to the old names? If any of them are simple renames, maybe we can reduce the differences between our versions.

Reply:
As said before I would prefer two different API's and then it would be weird to have alias with old ImageMagick 6 names.

Reply:
This will become aliases.